src/utils.ts

```diff
@@ -0,0 +1,697 @@
+// Dependencies
+import chalk from 'chalk';
+import fs from 'fs';
+import path from 'path';
+import Fuse from 'fuse.js';
+import ErrorWithoutStack from './error-without-stack';
+
+// Interface declarations
+interface InputObject {
+	command: string;
+	data?: string | number;
+	[index: string]: boolean | string | number | undefined;
+}
+
+// Interface declarations
+export interface AppSettings {
+	name: string | null;
+	packageName: string | null;
+	version: string | null;
+}
+
+export interface CommandSpec {
+	data: {
+		accepts?: string[];
+		description: string;
+		ignoreFlagsAndOptions?: boolean;
+		required?: boolean;
+		type?: string;
```

Comment:
We only support `integer` and `float` right now. Therefore, asking for any `string` value is not restrictive enough.

```suggestion
		type?: 'integer' | 'float';
```